stream-compositions/services/legal-entity-composition-service/pom.xml

```diff
@@ -1,160 +1,139 @@
-                <skipDefaultInterface>true</skipDefaultInterface>
-              </configOptions>
-            </configuration>
-          </execution>
-          <execution>
-            <id>generate-legal-entity-integration-client-code</id>
-            <phase>generate-sources</phase>
-            <goals>
-              <goal>generate-webclient-embedded</goal>
-            </goals>
-            <configuration>
-              <inputSpec>
-                ${project.basedir}/../../api/integrations-api/legal-entity-integration-api/src/main/resources/service-api-v2.yaml
-              </inputSpec>
-              <apiPackage>com.backbase.stream.compositions.legalentity.integration.client
-              </apiPackage>
-              <modelPackage>com.backbase.stream.compositions.legalentity.integration.client.model
-              </modelPackage>
-              <additionalProperties>useWithModifiers=true</additionalProperties>
-              <configOptions>
-                <reactive>true</reactive>
-              </configOptions>
-            </configuration>
-          </execution>
-          <execution>
-            <id>generate-product-composition-client-code</id>
-            <phase>generate-sources</phase>
-            <goals>
-              <goal>generate-webclient-embedded</goal>
-            </goals>
-            <configuration>
-              <inputSpec>
-                ${project.basedir}/../../api/service-api/product-composition-api/src/main/resources/service-api-v2.yaml
-              </inputSpec>
-              <apiPackage>com.backbase.stream.compositions.product.client</apiPackage>
-              <modelPackage>com.backbase.stream.compositions.product.client.model</modelPackage>
-              <additionalProperties>useWithModifiers=true</additionalProperties>
-              <configOptions>
-                <reactive>true</reactive>
-              </configOptions>
-
-            </configuration>
-          </execution>
-        </executions>
-      </plugin>
+    <name>Stream :: Compositions :: Services :: Legal Entity</name>
 
+    <dependencies>
+        <dependency>
+            <groupId>org.projectlombok</groupId>
+            <artifactId>lombok</artifactId>
+            <scope>provided</scope>
+        </dependency>
+        <!-- Streams dependencies -->
+        <dependency>
+            <groupId>com.backbase.stream.compositions.events</groupId>
+            <artifactId>legal-entity-events</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>com.backbase.stream.compositions.events</groupId>
+            <artifactId>product-events</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>com.backbase.stream</groupId>
+            <artifactId>legal-entity-core</artifactId>
+            <version>${project.version}</version>
+            <exclusions>
```

Comment:
legal-entity-core doesn't have sleuth dependency anymore, so this exclusion is redundant, perhaps you are building the composition modules testing without building the legal-entity-core before

Reply:
Perhaps there are some other transitive dependencies... anyway, I will fix this after the upgrade. In case if it's the only point of check